src/Traverse.js

```diff
@@ -1,10 +1,10 @@
 /* @flow */
 
-import type Element from '../../Element';
-import type Node from '../../Node';
-import type Token from '../../Token';
+import type Element from './elements/Element';
+import type Node from './elements/Node';
+import type Token from './elements/Token';
 
-export default class ElementSearchIndex {
+export default class Traversal {
```

Comment:
In dom analogues api's called in same manner, http://www.w3.org/TR/ElementTraversal/#interface-elementTraversal, https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelectorAll
https://api.jquery.com/category/traversing/